cmd/asciigraph/main.go

```diff
@@ -39,6 +44,42 @@ func main() {
 	flag.BoolVar(&enableRealTime, "r", enableRealTime, "enables `realtime` graph for data stream")
 	flag.IntVar(&realTimeDataBuffer, "b", realTimeDataBuffer, "data points `buffer` when realtime graph enabled, default equal to `width`")
 	flag.Float64Var(&fps, "f", fps, "set `fps` to control how frequently graph to be rendered when realtime graph enabled")
+	flag.Func("sc", "`series color` of the plot", func(str string) error {
+		if c, ok := asciigraph.ColorNames[str]; !ok {
+			return errors.New("unrecognized seriesColor")
+		} else {
+			seriesColor = c
+			return nil
+		}
+	})
+
+	flag.Func("cc", "`caption color` of the plot", func(str string) error {
+		if c, ok := asciigraph.ColorNames[str]; !ok {
+			return errors.New("unrecognized seriesColor")
```

Comment:
should this be "unrecognized caption color"?

Reply:
good catch! :+1: Fixed.